app/views/HowTaxIsSpentView.scala.html

```diff
@@ -35,7 +35,7 @@
 
     <h1 class="heading-xlarge">@messages("ats.howYourTaxWasSpent.heading")</h1>
 
-    <p>@Html(messages("ats.howYourTaxWasSpent.subheading", taxYear.toString, (taxYear+1).toString))</p>
+    <p>@Html(messages("ats.howYourTaxWasSpent.subheading", (taxYear-1).toString, taxYear.toString))</p>
```

Comment:
@vimalsuresh @anujakhare will need to go through the service to ensure the year is correct on every PAYE and SA page